TACC/Core-Styles
# enhance: accent color in s-footer

## Overview / Changes

- Make `s-footer` a dark context element.
- Use accent color on hover over link in `s-footer`.

> [!IMPORTANT]
> **Side Effect**: Headings in a footer will become white.
>
> <details><summary>Example</summary>
>
> | before | after |
> | - | - |
> | <img width="920" alt="before" src="https://github.com/user-attachments/assets/03bfd4bd-6a50-4fa5-8e12-a4af4bd0fb61"> | <img width="920" alt="after" src="https://github.com/user-attachments/assets/d98b5d1d-ba90-4b08-9a89-5e70abfb79cb"> |
>
> </details>

## Related

- required by https://github.com/TACC/tup-ui/pull/476

## Testing

1. Open http://localhost:3000/components/detail/s-footer.
2. Verify link color matches text color.
3. Hover over links.
4. Verify hover color is `#86aeff`.

## UI

https://github.com/user-attachments/assets/250ca050-d7e4-46bb-92f5-2475de289ab0

https://github.com/user-attachments/assets/a135d46f-c8b4-49ef-91e1-dcec07522013
